src/app/components/admin-components/admin/admin.component.ts

```diff
@@ -0,0 +1,42 @@
+import {Component, OnInit} from '@angular/core';
+import {ActivatedRoute} from '@angular/router';
+import {CategoryStatsService} from '@app/_services/api/admin/category-stats.service';
+import {QuestionCountService} from '@app/_services/api/admin/question-count.service';
+import {QuestionCount} from '@app/_models/question_counts';
+import {CategoryStats} from '@app/_models/category_stats';
+import {UserStatsService} from '@app/_services/api/user-stats.service';
+import {UserStats} from '@app/_models';
+
+@Component({
+    selector: 'app-admin',
+    templateUrl: './admin.component.html',
+    styleUrls: ['./admin.component.scss']
+})
+export class AdminComponent implements OnInit {
+    questionAPIData: QuestionCount[];
+    categoryAPIData: CategoryStats[];
+    userStatAPIData: UserStats[];
+
+    constructor(private categoryStatsService: CategoryStatsService,
+                private questionCountService: QuestionCountService,
+                private userStatsService: UserStatsService) {
+    }
+
+    ngOnInit(): void {
+        this.questionCountService
```

Comment:
This expanded form reduces the readability of the code. Please keep it in one line.

Reply:
Sorry, I am actually not sure what you are talking about here. Do you mean the empty constructors, or do you mean how I subscribe the variables? Need more context so I can fix this. The multi line empty constructors is how webstorm auto formats it, and the multi line subscribe is how I saw it done in many other components.